<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>morphology</class>
 <widget class="QWidget" name="morphology">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="structureElementWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>241</width>
     <height>161</height>
    </rect>
   </property>
   <widget class="QRadioButton" name="rectSE">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>35</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>rectangle</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="crossSE">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>36</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Cross</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ellipseSE">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>37</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Oval</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sets the core shape</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>113</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sets the number of iterations</string>
    </property>
   </widget>
   <widget class="QSlider" name="iterationSlider">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>133</y>
      <width>160</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="iterationSpinBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>133</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>62</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Set the core size</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>82</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Col</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="kernelColumnSpinBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>79</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>82</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Row</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="kernelRowSpinBox">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>78</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="morphologyWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>179</y>
     <width>241</width>
     <height>141</height>
    </rect>
   </property>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>67</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Open oper</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>67</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Closed oper</string>
    </property>
   </widget>
   <widget class="QPushButton" name="erosionButton">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>27</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Corrosion calc</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dilationButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>27</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Expansion oper</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hatButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>107</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Hat oper</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blackHatButton">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>107</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Black Hat Oper</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Perform morphological operations</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
